docs/features.md

```diff
@@ -1,20 +1,61 @@
 # Mainnet Feature Requirements
 
-## Nodes
-### As a Node Operator, I want to install software from binary artifacts or a Docker image
+## Node
+### As a Node perator, I want to install software from binary artifacts or a Docker image
+##### test: not available 
```

Comment:
those lines shoule not be here, those are part of `acceptance criteria` 
so only `###` level will suffice